modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -54,51 +54,69 @@ $ vim bmh.yaml
 [source,yaml]
 ----
 ---
+apiVersion: v1
+ kind: Secret
+ metadata:
+ name: openshift-worker-<num>-network <1>
+ type: Opaque
+ stringData:
+ nmstate: |
+  routes:
+   config:
+   - destination: 0.0.0.0/0
+    next-hop-address: <server ip address> <2>
```

Comment:
nit: for all user replaceable variables, use underscores. This makes it easier for the user to double-click to select: `<server_ip_address>`.